client/settings/phone-input/index.tsx

```diff
@@ -70,14 +70,29 @@ const PhoneNumberInput = ( {
 			}
 		};
 
+		let onlyCountries = {
```

Comment:
I think the name of the variable is a little misleading. Its name is `onlyCountries` and inside it there is an object with `onlyCountries`... What do you think about naming this variable `countries` or `phoneCountries` or something like this?